### ISSUE TITLE
Unable to set appDir

### ISSUE THREAD

**Issue by sanjibukai** [jemmyw/vscode-rails-fast-nav#4]
Hi,
I'm also experiencing the following error `command 'rails.fastNavigation' not found`
However, I tryed your extension since it seemed to manage the root app directory with the `appDir` option.
But I am unable to get your extension working (as all the others)..
My directory structure is still a regular one.
I just have a parent folder open, something like that:

```
my_project/
   + tools/
   + rails_app/
       + app/
       + config/
       + db/
   ...
```
So I open VS Code with that top folder, I'm running in the terminal `code my_project`

I tryed many combinaison of the option, eg. `"rails.appDir": "rails_app/app"`, `my_project/rails_app/app` (in fact all the combination with and without the parent and the app folder).
I tried with folder setting, workspace setting and even user setting...

I wonder why, all other plugins do not work too and even do not have the option at all..

**Comment by jemmyw** [comment]
Hi @sanjibukai, thanks for the report. This should be fixed in the latest version, 1.2.1.

I don't believe you even need to set `rails.appDir` any more, the presence of rails is detected using **/bin/rails and the specific rails project per file is found by recursing up the dir tree.

Cheers,
Jeremy

**Comment by sanjibukai** [comment]
Indeed it works now!
Thank you!
I'm used to not be bothered with some basic configuration things and just getting things _just working_ with convention over configuration stuffs...
